### PR TITLE
add the ability to toggle light and dark mode for themes

### PR DESCRIPTION
**Pull request by kelseythejackson**
Adds dark mode for all themes

### CODE REVIEW

File: docs/pages/experimental/themer.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ toc: false
     <a href="/">{% include 'logo.njk' %}</a>
     <wa-select name="theme" label="Theme" value="default">
       <wa-option value="default">Default</wa-option>
-      <wa-option value="classic">Classic</wa-option>
+      <wa-option value="default">Classic</wa-option>
```
Comment (KonnorRogers) [inline docs/pages/experimental/themer.md]: Any reason this was changed from "default" to "classic", when they're two different themes? Was this an accident?

**Review by KonnorRogers — COMMENTED** [review]
Looks good, just 1 question on themes, and then should be good to go

File: docs/pages/experimental/themer.md (hunk 1 above)
Comment (kelseythejackson) [reply docs/pages/experimental/themer.md]: Ooh, that's a slip by me, I'll fix that

Comment (kelseythejackson) [reply docs/pages/experimental/themer.md]: Okay, that's been updated

**Pull request merged**